README.md

```diff
@@ -50,6 +50,26 @@ python setup.py install
 ```
 
 
+## Docker
+
+Build docker image
+```bash
+docker build -t px-ldap-replace .
+```
+
+Prepare config files
+```bash
+mkdir .config
```

Comment:
Using a separate directory is a matter of taste.
Better tell the users, that `~/.config/px-ldap-replace.yaml` can also be used.